src/Console/IconSetConfig.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace BladeUI\Icons\Console;
+
+use Illuminate\Support\Str;
+
+class IconSetConfig
+{
+    public string $name;
+    public string $inputFilePrefix = '';
+    public string $outputFilePrefix = '';
+    private string $svgSourcePath;
+    private string $svgTempPath;
+    private string $svgDestinationPath;
+
+    private function __construct(string $name)
+    {
+        $this->name = $name;
+    }
+
+    public static function create(string $name): self
+    {
+        return new self($name);
+    }
+
+    public function setInputFilePrefix(string $filePrefix): self
+    {
+        $this->inputFilePrefix = $filePrefix;
+        return $this;
+    }
+
+    public function setOutputFilePrefix(string $filePrefix): self
+    {
+        $this->outputFilePrefix = $filePrefix;
+        return $this;
+    }
+
+    public function setSourcePath(string $svgSourcePath): self
+    {
+        $this->svgSourcePath = $svgSourcePath;
+        return $this;
+    }
+
+    public function getSourceFilePath(string $iconFileName): string
+    {
+        return $this->svgSourcePath . DIRECTORY_SEPARATOR . $this->name . DIRECTORY_SEPARATOR . $iconFileName;
```

Comment:
This is where we'd need to make adjustments for Icon packs where the "main" set are in a root folder.

Reply:
This is also resolved by using laravel's `Filesystem::files()` method to get a list of files. Given that this method returns a list of `\Symfony\Component\Finder\SplFileInfo` I removed this method from `IconSetConfig` all together.